apps/mobile/src/navigation/MainTabNavigator/PostIcon.tsx

```diff
@@ -1,18 +1,25 @@
-import Svg, { Path, SvgProps } from 'react-native-svg';
+import Svg, { G, Path, SvgProps } from 'react-native-svg';
 
 type Props = {
   color: string;
   size?: number;
 } & SvgProps;
 
 export const PostIcon = ({ color, size = 24, ...props }: Props) => {
+  const height = size;
+  const originalHeight = 24;
+  const scale = height / originalHeight;
+  const width = height;
+
   return (
```

Comment:
if we define a fixed viewBox here we should be able to change the size of the svg by just using width and height attributes. see `TwitterIcon` as an example